### DISCUSSION TITLE
Student Verification

### DISCUSSION THREAD

**Discussion by Nazchanel** [community/community#131887] [GitHub Education]
*So I am having major trouble with account verification.*

I submit my college student ID and am running into the following issue:

Your application cannot be reviewed until you fix the following:

Please ensure that your academic affiliation document contains your last name exactly as it appears in your [GitHub billing information](https://github.com/settings/billing/payment_information). If you have only a single legal name, enter it in both the first and last name fields of your GitHub billing information.

My hypothesis is because on my ID my name is shows as [First Name] [Middle Initial] [Last Name].

The billing information does not support a middle name or initial. I added my middle initial to my Last Name and it still didn't work. Can someone explain a work around so it detects my last name?

**Comment by samus-aran** [comment]
I'll be closing this discussion due to lack of recent activity. We can confirm that you have since been successful - enjoy your journey through GitHub Education.
